device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
+        "components": [],
+        "modules": [],
+        "fans": [
+            {
+                "name": "Fan 1",
+                "direction": "intake",
+                "tolerance": 20
```

Comment:
the fan tolerance has been updated to 50 to cover all the platforms and all the scenarios.

Reply:
fixed